examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -3,20 +3,21 @@
  * Licensed under the MIT License.
  */
 
-import isEqual from "lodash.isequal";
 import React, { useEffect, useState } from "react";
-
+import isEqual from "lodash.isequal";
 import { externalDataServicePort } from "../mock-external-data-service-interface";
-import type { IAppModel, TaskData } from "../model-interface";
+import type { IAppModel, ITaskList, TaskData } from "../model-interface";
 
 /**
  * Helper function used in several of the views to fetch data form the external app
  */
 async function pollForServiceUpdates(
 	externalData: Record<string, unknown>,
 	setExternalData: React.Dispatch<React.SetStateAction<Record<string, unknown>>>,
+	setFetchingExternalData: React.Dispatch<React.SetStateAction<boolean>>,
```

Comment:
`pollForServiceUpdates` function is only for debug data, so I would remove setting state in here.